hazelcast-client/src/main/java/com/hazelcast/client/console/ClientConsoleApp.java

```diff
@@ -386,8 +386,6 @@ public void run() {
             handleAddListener(args);
         } else if (first.equals("m.removeMapListener")) {
             handleRemoveListener(args);
-        } else if (first.equals("m.unlock")) {
-            handleMapUnlock(args);
```

Comment:
Can you justifly the removal? To me it looks like you're deleting functionality.

Reply:
Nice catch :)